### ISSUE TITLE
Usage with Jest 27.0.1?

### ISSUE THREAD

**Issue by karlhorky** [ikatyang/dts-jest#404]
Hi @ikatyang ! 👋 Hope that you are well!

I'm currently upgrading `babel-jest`, `jest`, `jest-worker`, and `ts-jest` all to version `27.0.1`, and most things are working now, but `dts-jest@23.3.0` fails in my tests with the following error:

```
FAIL dts src/__tests__/dts-jest/003-literals-unions.test.ts
  ● Test suite failed to run

    TypeError: Cannot read property '_dts_jest_' of undefined

      at Object.exports.transform [as process] (../../../../../node_modules/dts-jest/lib/transform.js:17:84)
      at ScriptTransformer.transformSource (../../../../../node_modules/@jest/transform/build/ScriptTransformer.js:612:31)
      at ScriptTransformer._transformAndBuildScript (../../../../../node_modules/@jest/transform/build/ScriptTransformer.js:758:40)
      at ScriptTransformer.transform (../../../../../node_modules/@jest/transform/build/ScriptTransformer.js:815:19)
```

Is `dts-jest` failing with v27?

**Comment by karlhorky** [comment]
Ok, I poked around a bit and opened this minimal PR, which appears to work on my system:

https://github.com/ikatyang/dts-jest/pull/405

**Comment by ikatyang** [comment]
Fixed by #405.
